pkg/metrics/crimetrics/crimetrics.go

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package crimetrics
+
+import (
+	"github.com/cilium/tetragon/pkg/metrics"
+	"github.com/cilium/tetragon/pkg/metrics/consts"
+)
+
+var (
+	CriResolutionsTotal = metrics.MustNewCounter(
+		metrics.NewOpts(
+			consts.MetricsNamespace,
+			"cri_cgidmap", "resolutions_total",
+			"number of total cgidmap CRI resolutions", nil, nil, nil),
```

Comment:
maybe you could explicit what `cgidmap` stands for? For someone reading these metrics out of the blue

Reply:
I've explained it better, but I'm still unsure if it helps. We need docs for this, but that's another PR.